apps/frontend/src/hooks/__tests__/algolia.test.tsx

```diff
@@ -1,31 +1,22 @@
-import {
-  algoliasearchLite,
-  SearchClient,
-  SearchIndex,
-} from '@asap-hub/algolia';
-import { ALGOLIA_APP_ID, ALGOLIA_INDEX } from '@asap-hub/frontend/src/config';
+import { ALGOLIA_INDEX } from '@asap-hub/frontend/src/config';
 import { renderHook } from '@testing-library/react-hooks';
 import { RecoilRoot } from 'recoil';
+import { algoliaSearchClientFactory } from '@asap-hub/algolia';
 
 import { useAlgolia } from '../algolia';
 import { Auth0Provider, WhenReady } from '../../auth/test-utils';
 
-var mockInitIndex: jest.MockedFunction<SearchClient['initIndex']>;
+var mockAlgoliaSearchClientFactory: jest.MockedFunction<
```

Comment:
Can this be a let?

Reply:
actually it cannot, jest is a bit funny about hoisting